### PR TITLE
YL-252 - Feature/candidate a11y report

### PR DESCRIPTION
**Pull request by sufyankhatri**
All of these changes are already approved. We need these changes in production that's why these changes need to merged into latest-pdf
https://spottedzebra.atlassian.net/browse/YL-252

### CODE REVIEW

File: components/molecules/CandidateReportHeaderMobile/CandidateReportHeaderMobile.tsx [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import './CandidateReportHeaderMobile.scss';
 import React, { FC } from 'react';
-import { Link } from 'react-router-dom';
+//TODO - YL-182
+// import { Link } from 'react-router-dom';
```
Comment (StefanosL) [inline components/molecules/CandidateReportHeaderMobile/CandidateReportHeaderMobile.tsx:4]: do we need comments in this PR?

**Review by StefanosL — APPROVED** [review]
I assume those comments were from before?

Can you please remove all unused code? (couldnt find the ticket YL-182)

File: components/molecules/CandidateReportHeaderMobile/CandidateReportHeaderMobile.tsx (hunk 1 above)
Comment (sufyankhatri) [reply components/molecules/CandidateReportHeaderMobile/CandidateReportHeaderMobile.tsx:4]: Yeah, I think these comments should be here until we figure out the root cause of this issue which is discussed here https://spottedzebra.atlassian.net/browse/YL-182

Comment (mljubic-biz) [reply components/molecules/CandidateReportHeaderMobile/CandidateReportHeaderMobile.tsx:4]: There is todo comment with task name so for me it is fine.

**Review by mljubic-biz — APPROVED** [review]

File: components/molecules/CandidateReportHeaderMobile/CandidateReportHeaderMobile.tsx (hunk 1 above)
Comment (mljubic-biz) [reply components/molecules/CandidateReportHeaderMobile/CandidateReportHeaderMobile.tsx:4]: Maybe just clearer comment.

**Pull request merged**